jooby-netty/src/main/java/org/jooby/internal/netty/NettyResponse.java

```diff
@@ -184,11 +189,28 @@ public void send(final FileChannel channel, final long offset, final long count)
     rsp.headers().set(headers);
     ChannelHandlerContext ctx = this.ctx;
     ctx.channel().attr(NettyRequest.NEED_FLUSH).set(false);
+
+    // add chunker
+    ChannelPipeline pipeline = ctx.pipeline();
+    if (pipeline.get("chunker") == null) {
```

Comment:
We need `chunker` only if we are doing SSL, right? We should probably test that and add when necessary.